bagitobjecttransfer/bagitobjecttransfer/settings/base.py

```diff
@@ -162,3 +163,65 @@
 CLAMAV_ENABLED = config('CLAMAV_ENABLED', cast=bool, default=True)
 CLAMAV_HOST = config('CLAMAV_HOST', default='clamav')
 CLAMAV_PORT = config('CLAMAV_PORT', cast=int, default=3310)
+
+# Pipeline configuration
+
+STATICFILES_STORAGE = 'pipeline.storage.PipelineManifestStorage'
+
+STATICFILES_FINDERS = (
+    'django.contrib.staticfiles.finders.FileSystemFinder',
+    'django.contrib.staticfiles.finders.AppDirectoriesFinder',
+    'pipeline.finders.PipelineFinder',
+)
+
+# create separate minified stylesheets and javascript files for each app
+PIPELINE = {
+    'PIPELINE_ENABLED': False,
+    'YUGLIFY_BINARY': os.path.join(BASE_DIR, 'node_modules/.bin/yuglify'),
+    'STYLESHEETS': {
+        'caais_styles': {
+            'source_filenames': (
+                'caais/css/*.css',
+            ),
+            'output_filename': 'caais/css/min.css',
+            'extra_context': {
+                'media': 'screen,projection',
+            },
+        },
+        'recordtransfer_styles': {
+            'source_filenames': (
+                'recordtransfer/css/*.css',
+            ),
+            'output_filename': 'recordtransfer/css/min.css',
+            'extra_context': {
+                'media': 'screen,projection',
+            },
+        },
+    },
+    'JAVASCRIPT': {
+        'caais_js': {
+            'source_filenames': (
+                'caais/js/*.js',
+            ),
+            'output_filename': 'caais.js',
+        },
+        'recordtransfer_base_js': {
+            'source_filenames': (
+                'recordtransfer/js/base/*.js',
+            ),
+            'output_filename': 'recordtransfer/js/base_min.js',
+        },
+        'recordtransfer_dropzone_js': {
+            'source_filenames': (
+                'recordtransfer/js/dropzone/*.js',
+            ),
+            'output_filename': 'recordtransfer/js/dropzone_min.js',
+        },
+    }
+}
+
+MIDDLEWARE.extend([
+    'django.middleware.gzip.GZipMiddleware',
+    'pipeline.middleware.MinifyHTMLMiddleware',
+])
```

Comment:
You may as well add these directly to the MIDDLEWARE setting, rather than extending the list here. Makes sense to have it all in one place

Reply:
Addressed by [a598d10](https://github.com/NationalCentreTruthReconciliation/Secure-Record-Transfer/pull/186/commits/a598d10a5e421a11ed5fcf9f0ef4a9174a8ca899)